### DISCUSSION TITLE
Account flagofft

### DISCUSSION THREAD

**Discussion by kennykentola** [vercel/community#4044] [Account Issue] [answered]
why is my account block,and suspended please am new here

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
